io/snmp/snmp.html

```diff
@@ -42,10 +82,16 @@
         color: "YellowGreen",
         defaults: {
             host: { value: "127.0.0.1" },
+            version: { value: "v1", required: true },
+            timeout: { value: 5 },
             community: { value: "public" },
-            version: { value: "1", required: true },
+            username: { value: "" },
+            auth: { value: "noAuthNoPriv", required: true },
+            authprot: { value: "None", required: true },
+            privprot: { value: "None", required: true },
+            authkey: { value: "" },
```

Comment:
Any credentials need to be handled as credentials and not general variables - see https://nodered.org/docs/creating-nodes/credentials (ie add a credential section like the defaults one but for the items that should be secret.)